Eevee.Sleep.Bot/Models/InGameAnnouncementDetailModel.cs

```diff
@@ -0,0 +1,19 @@
+using JetBrains.Annotations;
+using MongoDB.Bson.Serialization.Attributes;
+
+namespace Eevee.Sleep.Bot.Models;
+
+[BsonIgnoreExtraElements]
+public record InGameAnnouncementDetailModel: InGameAnnouncementMetaModel {
+    [UsedImplicitly]
+    public required string Content { get; init; }
+
+    [UsedImplicitly]
+    public required string ContentHash { get; init; }
+
+    [UsedImplicitly]
+    public required string Updated { get; init; }
+
+    [UsedImplicitly]
+    public required string RecordCreated { get; init; }
```

Comment:
Should be in type of `DateTime` with timezone info of `UTC`
Remember to store it in UTC and suffix this property with `Utc`